Makefile.mintlib

```diff
@@ -1,8 +1,8 @@
 include VARS
 
 PACKAGE_NAME=mintlib
-VERSION=Git-20200504
-VERSIONPATCH=-macos
+VERSION=Git-20220821
+VERSIONPATCH=
```

Comment:
You are removing the specific macOS patch, which was applied by setting "VERSIONPATCH". (See the content of "archive/mintlib-Git-20200504-macos.patch.bz2".)

Has this become obsolete respectively is it integrated in the newer 20220821 version?
If the patch is not necessary anymore, could you please remove the file "archive/mintlib-Git-20200504-macos.patch.bz2" too?

Reply:
Currently I'm unable to build it, as I get compiler crash... but this might be because of a defective cross-compiler 🫤
I'm on a M1... are you running on an Intel Mac?

```
m68k-atari-mint-gcc -B -B/opt/cross-mint/m68k-atari-mint/bin/ -B/opt/cross-mint/m68k-atari-mint/lib/ -isystem /opt/cross-mint/m68k-atari-mint/include -isystem /opt/cross-mint/m68k-atari-mint/sys-include -Wall -Werror -Wstrict-prototypes -Wmissing-prototypes -Wold-style-declaration -Wold-style-definition -O2 -fomit-frame-pointer -fgnu89-inline -nostdinc -I../argp -I.. -I../include -I../mintlib -I../stdlib -I/opt/cross-mint/lib/gcc/m68k-atari-mint/4.6.4/include -I/opt/cross-mint/lib/gcc/m68k-atari-mint/4.6.4/include-fixed -DHAVE_CONFIG_H -D_LIBC -D_REENTRANT -Wp,-MD,../.deps/argp-fmtstream.P -c ../argp/argp-fmtstream.c -o argp-fmtstream.o
../argp/argp-fmtstream.c: In function '__argp_fmtstream_update':
../argp/argp-fmtstream.c:427:1: internal compiler error: Illegal instruction: 4
Please submit a full bug report,
with preprocessed source if appropriate.
See <http://gcc.gnu.org/bugs.html> for instructions.
```

Reply:
On my old Intel x86_64 Mac I successfully tested `make`. 👍🏻
So the remaining issues are still building on/for M1 Macs. But that's not one of your topics.